src/Missions.java

```diff
@@ -0,0 +1,25 @@
+import java.util.Scanner;
+
+public class Missions {
+    public static Scanner scanner = new Scanner(System.in);
+
+    public static int inputNumber(){
+        System.out.println("Enter a number:");
+        return scanner.nextInt();
+    }
+
+    public static int maxNumber(){
+        int number = inputNumber();
+        int max = Integer.MIN_VALUE;
+
+        while (number != 0) {
+            max = Math.max(max, number);
+            number = inputNumber();
+        }
+        return max;
+    }
```

Comment:
like its annoying because the good thing to do is
function that input numbers array
function that return max of them

but i dont want u to use array

SO u dont need to change but what a bulbul